bigbluebutton-html5/imports/ui/components/video-provider/fullscreen-button/component.jsx

```diff
@@ -29,7 +29,7 @@ const FullscreenButtonComponent = ({
 }) => {
   const formattedLabel = intl.formatMessage(
     intlMessages.fullscreenButton,
-    ({ 0: elementName ? elementName.toLowerCase() : '' }),
+    ({ 0: elementName || '' }),
```

Comment:
This will cause the tooltip for the presentation fullscreen button to no longer display in sentence case. 

Is this still something that @tylercopeland  wants?
https://github.com/bigbluebutton/bigbluebutton/issues/6299

Reply:
That issue is specifically for static strings that we create. We don't want to be manipulating user provided data.